rel-eng/tito.props

```diff
@@ -963,6 +963,7 @@ disttag = .el8
 # don't whitelist foreman-release here, it's already in foreman-nightly-el8
 whitelist =
   ansible-collection-theforeman-foreman
+  jsoncpp
```

Comment:
I think this should be in the katello-nightly-el8 tag since it will be used to build qpid_proton packages ?